syntax = "proto3";

message EnrollRequest {
    string username = 1;
    string user_encoded_video = 2;
}

message EnrollResponse {
    string message = 1;
    string data = 2;
    string username = 3;
    string enrollment = 4; 
}

service FaceRecognitionEnrollmentService {
    rpc FaceRecognitionEnroll (EnrollRequest) returns (EnrollResponse);
}